pkg/descheduler/descheduler.go

```diff
@@ -107,6 +109,15 @@ func Run(ctx context.Context, rs *options.DeschedulerServer) error {
 		return nil
 	}
 
+	// Add k8s compatibility warnings to logs
+	serverVersion, serverErr := rs.Client.Discovery().ServerVersion()
+	if serverErr != nil {
+		klog.V(1).InfoS("Warning: Get Kubernetes server version fail")
```

Comment:
Same here. InfoS and then followed by Warning in text. Might as well use klog Warning?

Reply:
the same reason as above. in the https://github.com/kubernetes-sigs/descheduler/blob/master/pkg/descheduler/descheduler.go, all of them are using `klog.V(1).InfoS("Warning:`. so i just align.
@damemi  @a7i  please confirm is it i need to replace all the `klog.V(1).InfoS("Warning:` to `klog.Warning()`.

Reply:
We don't need to update all cases of that in this PR, but we can start with just this change using `Warning()` instead of `InfoS`.

If you'd like, it would be nice to open another PR to find other instances of this that can be updated

Reply:
sure, i already change to `Warning()` instead of `InfoS`